addons/crm/wizard/crm_lead_to_opportunity_mass_views.xml

```diff
@@ -18,20 +18,20 @@
                 </group>
                 <label for="duplicated_lead_ids" string="Leads with existing duplicates (for information)" help="Leads that you selected that have duplicates. If the list is empty, it means that no duplicates were found" invisible="not deduplicate"/>
                 <group invisible="not deduplicate">
-                    <field name="duplicated_lead_ids" colspan="4" nolabel="1" readonly="1">
```

Comment:
Actually, you can regroup changes by module as well instead of doing 1commit/functionality . You can squash this commit with other ones of crm. Let us not use too many commits if it does not bring much value :)